docs/README.md

```diff
@@ -1,29 +1,30 @@
-# User Guide
+# Duke User Guide
```

Comment:
I see that you have actually renamed your ChatBot to Linus! It's a nice name! Might it be better to change your main class to Linus to avoid confusion with Duke?